src/main/java/InputParser.java

```diff
@@ -0,0 +1,24 @@
+public class InputParser {
```

Comment:
Good job on the InputParser class which seems versatile and can handle multiple types of commands